### PR TITLE
New jekyll cheat sheet

### PR DESCRIPTION
**Pull request by navneetzz**
**What does your Instant Answer do?**
My Instant answer provide reference to Jekyll a static site generator.

**What is the data source for your Instant Answer? (Provide a link if possible)**
Jekyllrb (https://jekyllrb.com/)

**Why did you choose this data source?**
Because it's official site of Jekyll.

**Are there any other alternative (better) data sources?**
No

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Web developers and bloggers will find it very useful.

**Where did you hear about DuckDuckHack? (For first time contributors)**
Social media sites.

jekyll

---
https://duck.co/ia/view/jekyll_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Jekyll Cheat Sheet](https://duck.co/ia/view/jekyll_cheat_sheet)

**Description**: A quick reference to Jekyll, a static site generator

**Example Query**: [jekyll cheat sheet](https://beta.duckduckgo.com/?q=jekyll cheat sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: Jekyllrb

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/jekyll_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/jekyll_cheat_sheet)*

File: share/goodie/cheat_sheets/json/jekyll.json [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                "key": "site.html_pages",
+                "val": "A list of all HTML Pages"
+            },
+            {
+                "key": "site.collections",
+                "val": "A list of all the collections"
+            },
+            {
+                "key": "site.data",
+                "val": "A list containing the data loaded from the files located in the _data folder"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/jekyll.json]: @navneetzz Lets remove the "A" from beginig of above val fields. They are a bit redundant

Comment (navneetzz) [inline share/goodie/cheat_sheets/json/jekyll.json]: @jophab "A" is more relevant then simply writing that sentence.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=jekyll cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/jekyll_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=jekyll cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/jekyll_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@navneetzz Thanks! This LGTM :+1:

**Pull request merged**